Jenkinsfile

```diff
@@ -22,20 +26,22 @@ pipeline {
                 sh 'mvn -B -V verify -Prun-its -Pci'
             }
         }
-        stage('Check Image Build Hook') {
-            when {
-                expression { env.IMG_BUILD_HOOKS != null }
-            }
+        stage('Load OCP Mappings') {
             steps {
-                echo "Check Image Build Hook"
+                echo "Load OCP Mapping document"
                 script {
-                    def jsonObj = readJSON text: env.IMG_BUILD_HOOKS
-                    if (env.GIT_URL in jsonObj) {
-                        echo "Build docker image"
-                        if (env.BRANCH_NAME in jsonObj[env.GIT_URL]) {
-                            img_build_hook = jsonObj[env.GIT_URL][env.BRANCH_NAME]
-                        } else {
-                            img_build_hook = jsonObj[env.GIT_URL]['default']
+                    def exists = fileExists ocp_map
+                    if (exists){
+                        def jsonObj = readJSON file: ocp_map
+                        if (bc_section in jsonObj){
+                            if (env.GIT_URL in jsonObj[bc_section]) {
+                                echo "Found BC for Git repo: ${env.GIT_URL}"
+                                if (env.BRANCH_NAME in jsonObj[bc_section][env.GIT_URL]) {
+                                    img_build_hook = jsonObj[bc_section][env.GIT_URL][env.BRANCH_NAME]
```

Comment:
I guess this should be "my_bc=....". The img_build_hook is abandoned as I see it.

Reply:
good catch, I was in a bit of a hurry when I submitted this PR